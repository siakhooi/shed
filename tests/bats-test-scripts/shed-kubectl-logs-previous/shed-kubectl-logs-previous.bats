setup(){
    load '../common-setup'
    common_setup
}
@test "shed-kubectl-logs-previous" {
    run shed-kubectl-logs-previous
    assert_success
    cat_expected "shed-kubectl-logs-previous-usage" | assert_output -
}
@test "shed-kubectl-logs-previous xxx yyy zzz" {
    run shed-kubectl-logs-previous xxx yyy zzz
    assert_success
    cat_expected "shed-kubectl-logs-previous-usage" | assert_output -
}
@test "shed-kubectl-logs-previous xxx" {
    run shed-kubectl-logs-previous xxx
    assert_failure 1
    cat_expected "shed-kubectl-logs-previous-xxx" | assert_output -
}
@test "shed-kubectl-apply hello-world; shed-kubectl-logs-previous hello-world" {
    skip 'Unable to use a TTY - input is not a terminal or the right kind of file'
    shed-kubectl-apply hello-world
    run bash -ic "echo 'exit' |shed-kubectl-logs-previous say-hello "
    assert_success
    cat_expected "shed-kubectl-logs-previous" | assert_output -
    shed-kubectl-delete hello-world
}
@test "shed-kubectl-apply hello-world; shed-kubectl-logs-previous hello-world busybox-1" {
    skip 'Unable to use a TTY - input is not a terminal or the right kind of file'
    shed-kubectl-apply hello-world
    run bash -ic "echo 'exit' |shed-kubectl-logs-previous say-hello busybox-1"
    assert_success
    cat_expected "shed-kubectl-logs-previous-container" | assert_output -
    shed-kubectl-delete hello-world
}
