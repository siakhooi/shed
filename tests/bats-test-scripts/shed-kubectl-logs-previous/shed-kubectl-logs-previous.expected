+ kubectl --context kube-dev-1 --namespace ns-application logs --previous busybox-looping
